app/picker.html

```diff
@@ -0,0 +1,422 @@
+            margin-right: 0.5rem;
+        }
+
+        .summary-container table {
+            width: 100%;
+            border-collapse: collapse;
+        }
+
+        .summary-container th, .summary-container td {
+            border: 1px solid #ddd;
+            padding: 8px;
+            text-align: left;
+        }
+
+        .summary-container th {
+            background-color: #f4f4f4;
+        }
+
+        .additional-buttons {
+        }
+
+        .additional-buttons .btn {
+            display: block;
+            padding: 10px;
+            margin-top: 10px;
+            text-align: center;
+            border-radius: 4px;
+            background-color: #28a745;
+            color: white;
+            text-decoration: none;
+            box-sizing: border-box;
+        }
+
+        .additional-buttons .btn:hover {
+            background-color: #218838;
+        }
+
+        .additional-buttons .btn-setup {
+            background-color: #007bff;
+        }
+
+        .additional-buttons .btn-setup:hover {
+            background-color: #0056b3;
+        }
+
+        .untracked {
+            background-color: #FFFFFF;
+        }
+
+        /* White, as requested */
+        .present {
+            background-color: #4CAF50;
+        }
+
+        /* A softer green */
+        .not-present {
+            background-color: #F44336;
+        }
+
+        /* A slightly muted red */
+        .other {
+            background-color: #2196F3;
+        }
+
+        /* A vibrant but not too bright blue */
+        .today {
+            border: 5px solid #FFC107;
+            font-weight: bold;
+            padding: 3px;
+        }
+
+        /* A golden yellow for emphasis */
+
+        @media (max-width: 600px) {
+            #calendar {
+                grid-template-columns: repeat(3, 1fr); /* For smaller screens, show fewer columns */
+            }
+
+            .btn {
+                padding: 0.25rem 0.5rem;
+                font-size: 0.8rem;
+            }
+        }
+
+        @media (max-width: 768px) {
+            .container {
+                padding: 0 10px;
+            }
+        }
+
+    </style>
+</head>
+<body>
+
+<div class="container">
+    <!-- All your content goes here -->
+
+    <div id="calendar-nav">
+        <button id="prev-month">Previous</button>
+        <span id="month-year"></span>
+        <button id="next-month">Next</button>
+    </div>
+
+    <div id="calendar"></div>
+
+    <div id="legend" class="legend">
+        <div class="legend-item">
+            <span class="legend-color untracked"></span> Untracked
+        </div>
+        <div class="legend-item">
+            <span class="legend-color present"></span> Work from Home
+        </div>
+        <div class="legend-item">
+            <span class="legend-color not-present"></span> In office
+        </div>
+        <div class="legend-item">
+            <span class="legend-color other"></span> Other (Untracked)
+        </div>
+    </div>
```

Comment:
Looks good. One suggestion is removing `Other (Untracked)` or changing the name to make it clearer what the difference is from `Untracked`.

```suggestion
    <div id="legend" class="legend">
        <div class="legend-item">
            <span class="legend-color untracked"></span> Untracked
        </div>
        <div class="legend-item">
            <span class="legend-color present"></span> Work from Home
        </div>
        <div class="legend-item">
            <span class="legend-color not-present"></span> In office
        </div>
    </div>
```